paddle/fluid/operators/compat/fused_elementwise_add.pbtxt

```diff
@@ -0,0 +1,50 @@
+type: "fused_elementwise_add"
+def {
+  inputs {
+    name: "X"
+  }
+  inputs {
+    name: "Y"
+  }
+  outputs {
+    name: "Out"
+  }
+  attrs {
+    name: "axis"
+    type: INT
+  }
+}
+extra {
+  attrs {
+    name: "x_data_format"
+    type: STRING
+  }
+  attrs {
+    name: "y_data_format"
+    type: STRING
+  }
+  attrs {
+    name: "alpha"
+    type: FLOAT
+  }
+  attrs {
+    name: "beta"
+    type: FLOAT
+  }  
+  attrs {
+    name: "Scale_x"
```

Comment:
```suggestion
    name: "scale_x"
```
All attributes should be lowercase. We must unify it, as there are currently too many duplicates with quantization scales.